src/actions/adminActions.js

```diff
@@ -5,28 +5,32 @@ import { notify } from '../utils';
 import { logoutUser } from './authActions';
 
 export const postEvent = (event) => async dispatch => {
```

Comment:
Does the postEvent action need to be async since we are already returning a promise that takes in an async function?

Reply:
The postEvent function simply returns a promise object that takes in an async function. The returning part should be synchronous I think.